.github/workflows/test.yml

```diff
@@ -53,6 +53,8 @@ jobs:
       - uses: actions/setup-go@v2.1.4
         with:
           go-version: 1.17
+      - name: install rocksdb
```

Comment:
Why not use a docker image? This will be run each time

Reply:
Creating a docker to test one package?

Reply:
Docker would especially help with the source install